### DISCUSSION TITLE
Roadmap Progress

### DISCUSSION THREAD

**Discussion by aydreeihn** [osTicket/roadmap#66] [General]
Just as a reminder, 2.0 Progress can be seen within the [Roadmap Project](https://github.com/orgs/osTicket/projects/2). We may not update it super frequently, but we are also very focussed on the actual development being done. This at least shows some of the items are being scoped out and worked on. A new Phase has been added to show what has been completed (Done). We were previously moving the ToDo's to the closed status, so some items that are complete may not necessarily show up in the timeline. Each item on the ToDo list has a writeup that was added as an Issue to the roadmap. The closed issues can still be seen within the list of issues, and those have been completed.

**Comment by RikkiMasters** [comment]
Hi Team, any progress updates? Curious to start playing with the new version.

Thanks

**Comment by JediKev** [comment]
@RikkiMasters 

Very good timing lol We quite literally just hired a dedicated Project Manager; among other things. This likely means that our Roadmap, etc. will be receiving regular/semi-regular updates as we further progress. Exciting things coming down the pipeline so stay tuned!

Cheers.

**Reply by waczny** [reply]
Hi,
what about the updates from the project manager?  Just curious. 
BR

**Reply by JediKev** [reply]
@waczny 

The PM is more behind the scenes (as well as our roadmap progress) but I can say that the progress is going splendidly well. At this point in time we are still on track to have a preview out this year. We are really happy with the front end redesigns and rewrites. We are chugging along with the backend rewrites, improving existing features, and adding some new features in the process. So far super good.

Cheers.

**Reply by mlipok** [reply]
> At this point in time we are still on track to have a preview out this year.

Finally.
Thanks for the info.

**Reply by waczny** [reply]
> At this point in time we are still on track to have a preview out this year
@JediKev I assume this is not the case anymore? I just ask because you said it.

Without being negative, I assume if there is no preview for anything there won't be a somewhat useable beta for 2026. 
I completely understand this is an opensource project with good people but limited resources. 
I don't fully understand why it's not possible to have a roadmap with some milestones for the public. This isn't a school project you guys are specialists and have/had a great software with the previous version and gathered a great community. 
I know from my own job how it is to stick to a timeline but it's not a big deal you at least won't have to justify why a timeline was not hit to the public :-)

**Reply by JediKev** [reply]
See here:
- https://github.com/osTicket/roadmap/discussions/36#discussioncomment-15177761

TLDR; Just be patient 🙂 Big news coming and there will 100% be a beta release 2026.

**Comment by TheMitelGuy** [comment]
Hello!

I hope the team is doing well!  Happy new year from Canada!

I had a quick question about the 2.0 release.  Understandably it's a long process with a completely new code base.  Just curious - is there any plans to have a (simple) migration from 1.8.x to 2.0?  I looked at the task list in GitHub, but I don't see any descriptions that would lead me to believe 2.0 will have a migration tool or a way to restore from a 1.8.x backup...  As I write this question, I'm actually trying to think what is more important, the actual tickets/cases and the data they contain or the user/customer (credential's) database?  I'm guessing that the latter would be the most important and the first (historical ticket data) would be secondary...   Anyways, just curious if that's the plan, to allow customers to go from 1.8.x to 2.x.y?

Thanks so much - I'm getting excited to see what 2.0 looks like!

**Reply by JediKev** [reply]
We will have an upgrade wizard like before so all legacy data will carry over. Exciting times ahead!

**Reply by TheMitelGuy** [reply]
Okay - perfect!  Thanks so much!

**Comment by waczny** [comment]
Will 2.0 support styles for smaller screens like phones/ tablets/ etc ?

**Reply by JediKev** [reply]
Of course, it will be fully modern and responsive (which includes mobile devices).

**Comment by TheMitelGuy** [comment]
I’m excited!

Thanks for the update!

Neil

**Reply by TheMitelGuy** [reply]
Hello everyone!

I hope this e-mail finds everyone well!

Just a quick question.  I've followed a couple of things that came out of this thread, but it just adds to my confusion.   I'm just curious - are we still on track to have a Beta version of OSTicket 2.0 this calendar year?  I believe that's where we last left off at the beginning of the calendar year (January of this year).  I'm getting more and more eager.  I want to trying integrating it between SAP Business One (for finance/accounting) and also our phone system (Mitel) contact centre (creating a ticket over the phone) and/or having a ticket number in order to get your call into the correct queue.  I don't think this will take too work much, but I obviously have to have something to test with - so I thought I would ask and just see where we are at!

Hope everyone is doing well and has a nice Remembrance Day tomorrow (Canada, UK, Australia and the other Commonwealth countries around the world).

**Comment by kcraftcode** [comment]
Hello @aydreeihn ,
How are you?
I m very interersted about this project.
Is there any opportunity to work with you in your team?

**Comment by RikkiMasters** [comment]
Any news?

**Reply by JediKev** [reply]
Read above; here is my latest comment on the subject:
- https://github.com/osTicket/roadmap/discussions/66#discussioncomment-13066159
